packages/radix-vue/src/VisuallyHidden/VisuallyHiddenInput.vue

```diff
@@ -28,7 +28,7 @@ const parsedValue = computed(() => {
   }
 
   // if object value
-  else if (typeof props.value === 'object' && !Array.isArray(props.value)) {
+  else if (typeof props.value === 'object' && props.value !== null) {
```

Comment:
We need `!Array.isArray(props.value)` to make sure this value is type object and not an array.